sys/shell/commands/sc_ccnl.c

```diff
@@ -170,10 +167,16 @@ int _ccnl_interest(int argc, char **argv)
         addr_len = gnrc_netif_addr_from_str(relay_addr, sizeof(relay_addr), argv[2]);
     }
 
+    memset(_int_buf, '\0', BUF_SIZE);
+    memset(_cont_buf, '\0', BUF_SIZE);
```

Comment:
without looking into `ccnl_send_interest` and `ccnl_wait_for_chunk`: wouldn't it be necessary to reset those buffers after each try (within the loop)?